recipe/meta.yaml

```diff
@@ -30,8 +30,7 @@ outputs:
       run:
         - cfgv >=2.0.0
         - identify >=1.0.0
-        - importlib_metadata  # [py<38]
-        - importlib_resources <5.3  # [py<37]
```

Comment:
the version specifier got removed here

it might make sense to just make the feedstock require 3.8+ instead ?

Reply:
I'm the pre-commit maintainer

there is harm, the latest version of `importlib-resources` subtly breaks pre-commit

Reply:
thanks... that is good input! ... hm, then either the feedstock can require python 3.8+ or disallow specific importlib-resources version ranges

Reply:
oh wait I just misread this entirely -- this is fine as is